pymonntorch/NetworkCore/Network.py

```diff
@@ -116,7 +122,7 @@ def clear_recorder(self, keys=None):
     def __repr__(self):
         neuron_count = torch.sum(torch.tensor([ng.size for ng in self.NeuronGroups]))
         synapse_count = torch.sum(
-            torch.tensor([sg.src.size * sg.dst.size for sg in self.SynapseGroups])
+            torch.tensor([sg.src.size * sg.dst.size for sg in self.SynapseGroups if (sg.src is not None and sg.dst is not None)])
```

Comment:
what is the purpose of if statement

Reply:
A SynapseGroup can be created without a src or dst population therefore calculating the number of synapses is futile.